images/openshift-enterprise-template-service-broker-operator.yml

```diff
@@ -2,17 +2,17 @@ name: openshift/openshift-enterprise-template-service-broker-operator
 
 owners:
 - ccpeng-team@redhat.com
-- jmontleo@redhat.com
+- jkim@redhat.com
 
 content:
   source:
-    dockerfile: build/Dockerfile
+    dockerfile: Dockerfile.rhel
     git:
       branch:
         fallback: master
         target: release-{MAJOR}.{MINOR}
       url: git@github.com:openshift/template-service-broker-operator.git
 
 from:
-  member: openshift-enterprise-base
+  member: openshift/ose-ansible-operator
```

Comment:
@joeldavis84 is still working on this image and it's a WIP -> https://github.com/openshift/ocp-build-data/blob/openshift-4.0/images/openshift-enterprise-ansible-operator.yml#L1
We cannot merge this until he's done with that image and it's actually available.

Reply:
The image is part of the current build process (incl. successful build for 0.159.0), I just forgot to un-WIP it yesterday but I did so just now.